AnkiDroid/src/main/java/com/ichi2/anki/Ease.kt

```diff
@@ -17,10 +17,21 @@ package com.ichi2.anki
 
 /**
  * [value] should be kept in sync with the [com.ichi2.anki.api.Ease] enum.
+ *
+ * @param value The so called value of the button. For the sake of consistency with upstream and our API
+ * the buttons are numbered from 1 to 4.
  */
 enum class Ease(val value: Int) {
     AGAIN(1),
     HARD(2),
     GOOD(3),
     EASY(4);
+
+    companion object {
+        /**
+         *The function associating to [value] the Ease whose value is [value].
+         * E.g. it sends 1 to [Again] and 4 to [Easy]
+         */
+        fun fromValue(value: Int) = Ease.entries[value - 1]
```

Comment:
The implementation assumes that the values in the enum are ordered. This is the case now, and tbh it is unlikely to change, but I prefer to avoid future human errors when possible.

```suggestion
        fun fromValue(value: Int) = entries.first { it.value == value }
```

Reply:
Currently, this is guaranteed by the documentation. https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.enums/-enum-entries.html But fine. I doubt this is in the loop where it would be an issue to iterate over four elements